mixer/template/authz/template.proto

```diff
@@ -0,0 +1,49 @@
+// Copyright 2017 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package authz;
+
+import "mixer/v1/config/descriptor/value_type.proto";
+import "mixer/v1/template/extensions.proto";
+
+option (istio.mixer.v1.template.template_variety) = TEMPLATE_VARIETY_CHECK;
+
+message Subject {
+  // The user.
+  string user = 1;
+  // Groups the subject belongs to.
+  repeated string groups = 2;
```

Comment:
`repeated`. This is the first template we have used repeated. repeated fields are not completely supported yet. I can make it work in 1/2 day (just need to add its procession in the bootstraping code (loading yaml and generating instance objects), the code for generating artifacts to be used by the adapter is already present). 

This should not block your adapter implementation thought. But unfortunately without change from me, you won't be able to test your adapter e2e with Mixer.

Reply:
That should be fine. We can keep this field repeated for now and write a e2e test after the implementation.